contrib/postgres_proxy/filters/network/source/BUILD

```diff
@@ -12,7 +12,7 @@ envoy_contrib_package()
 #package(default_visibility = ["//visibility:public"])
 
 # PostgresSQL proxy L7 network filter.
-# Public docs: docs/root/configuration/network_filters/postgres_proxy_filter.rst
+# Public docs: docs/root/configuration/listeners/network_filters/postgres_proxy_filter.rst
```

Comment:
with these links that are prefixed `docs/root` (ie repo relative links) im wondering if we should replace with links to the actual docs (as in links below)

Reply:
IMO, that change seems like it would be more useful to developers as text editing apps may automatically create a clickable link to see what's already labeled as public docs.

Reply:
yep, and the repo rst files are source files so not as easy/useful to read